x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -243,7 +244,7 @@ func lineToSynthEventFactory(typ string) func(bytes []byte, text string) (res *S
 		logp.Info("%s: %s", typ, text)
 		return &SynthEvent{
 			Type:                 typ,
-			TimestampEpochMicros: float64(time.Now().UnixNano() / int64(time.Millisecond)),
+			TimestampEpochMicros: float64(time.Now().UnixMicro()),
```

Comment:
I am not sure about the intended consequence of this change, I will leave it to @andrewvc who knows it better than me.

Reply:
This change is because timestamps calculated in that way would yield negative values, and thus not appear in my queries (I was filtering for events in the last X years).

`UnixMicro` is directly what we want, so it also makes the code a bit more concise, unless I'm missing any other behaviours from the `stdlib` here.

Reply:
I had no idea the `UnixMicro` method existed, As far as I'm concerned this is a nice improvement, so I'm +1 on fixing this here.